### DISCUSSION TITLE
Suitable as a DAM?

### DISCUSSION THREAD

**Discussion by ghost** [photoprism/photoprism#4057] [Q&A]
I'm looking for a Digital Asset Management solution for a car dealership business. Is Photoprism a suitable solution for this?
There are around 3000 cars sold each year, and each car can have a small gallery of images.

Bonus question: can I link the gallery of a given car to a record in a Baserow or Nocodb database?

**Comment by lastzero** [comment]
PhotoPrism is not a product management system by design - however it could be used to manage product images and then be extended with such functionality or be used in combination with a DAM that manages the product details such as price, category, location, etc.
